privval/file_test.go

```diff
@@ -349,3 +357,25 @@ func newProposal(height int64, round int32, blockID types.BlockID) *types.Propos
 		Timestamp: tmtime.Now(),
 	}
 }
+
+func BenchmarkFilePVSignVote(b *testing.B) {
+	tempDir := b.TempDir()
+	privVal, err := GenFilePV(filepath.Join(tempDir, "key_file"), filepath.Join(tempDir, "state_file"), "")
+	require.NoError(b, err)
+
+	// test vote
+	height, round := int64(10), int32(1)
+	voteType := tmproto.PrevoteType
+	randBytes := tmrand.Bytes(tmhash.Size)
+	blockID := types.BlockID{Hash: randBytes, PartSetHeader: types.PartSetHeader{}}
+	vote := newVote(privVal.Key.Address, 0, height, round, voteType, blockID)
+
+	b.ReportAllocs()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		if err := privVal.SignVote(context.Background(), "mychainid", vote.ToProto()); err != nil {
+			b.Fatal(err)
+		}
+		privVal.Reset()
```

Comment:
Not sure if it makes sense to include this line in the benchmark since we don't do this in running nodes.